src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,157 @@
 package core.basesyntax;
 
+import java.util.Collection;
```

Comment:
The import statement for 'java.util.Collection' is not necessary, as it is not being used in the code. Please remove it to keep the code clean. 🧹